### PR TITLE
organizationPage Updated

### PR DESCRIPTION
**Pull request by praveenaraoa**
Updated as per the requirements mentioned in issue #125 and issue #40

### CODE REVIEW

**Comment by russon77** [comment]
@praveenaraoa no need to comment out code, instead you should delete it. git remembers all changes, and leaving commented out code (without explanation) confuses more than it helps.

**Comment by praveenaraoa** [comment]
@russon77 Thank you :). I just removed the commented code in this case.

**Comment by Code4SocialGood** [comment]
The PR failed Travis CI build due to tslint error. Please run` npm run lint` to review and fix the lint errors.

**Comment by praveenaraoa** [comment]
sure. i will look into it. thanks

**Comment by praveenaraoa** [comment]
Tlint issues resolved

File: src/app/organization/create/organization-create.component.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<div class="container">
+
+\<div class="container">
```
Comment (russon77) [inline src/app/organization/create/organization-create.component.html]: accidental `\` character at hte beginning of this line

File: src/app/organization/create/organization-create.component.html [hunk 2]
```diff
@@ -23,136 +24,129 @@ <h3 class="center-align">Create Organization</h3>
...
-             <i class="material-icons tiny req-field pull-right">star</i>
+              
+             
+            <label for="name" class="form-label" >
+              Organization Name*
+            
             </label>
 
-            <input placeholder="" type="text" id="name" [ngClass]="{active: formPlaceholder.name}" formControlName="name" [ngClass]="{'validate':!organizationForm.controls['name'].valid}">
+            <input type="text" id="name" class="form-input" floating [ngClass]="{active: formPlaceholder.name}" formControlName="name" [ngClass]="{'validate':!organizationForm.controls['name'].valid}" >
```
Comment (russon77) [inline src/app/organization/create/organization-create.component.html]: two `ngClass` attributes should be consolidated

File: src/app/organization/create/organization-create.component.html (hunk 1 above)
Comment (praveenaraoa) [inline src/app/organization/create/organization-create.component.html]: Thank you. Removed \ character

File: src/app/organization/create/organization-create.component.html (hunk 2 above)
Comment (praveenaraoa) [inline src/app/organization/create/organization-create.component.html]: combined two ngclass attributes

**Comment by praveenaraoa** [comment]
Requested issues are addressed. Thank you russon77

**Comment by Code4SocialGood** [comment]
Create Organization page is to be deleted

**Pull request closed without merging**